src/views/create/index.tsx

```diff
@@ -124,8 +125,7 @@ const Create = (_: RouteComponentProps) => {
                                         network
                                     );
 
-                                    doContractDeploy({
-                                        network: stacksNetwork,
+                                    openContractDeploy({
```

Comment:
micro-stacks doesn't require network argument since it handles network updates itself i guess? right?

Reply:
yes, that is correct!